hibernate-core/src/test/java/org/hibernate/test/bytecode/enhancement/lazy/proxy/LazyGroupWithInheritanceTest.java

```diff
@@ -61,8 +79,8 @@ public void queryEntityWithAssociationToAbstract() {
 					//			The only viable solution I see would be to join to the "other side" and read the
 					//			version/discriminator[1].  But of course that means doing the join which is generally
 					//			what the application is trying to avoid in the first place
-					//expectedQueryCount.set( 1 );
-					expectedQueryCount.set( 4 );
+					expectedQueryCount.set( 1 );
```

Comment:
Is the effect of this PR, that `Order#customer` is eagerly fetched now or "bytecode initialized".

Reply:
It is a HibernateProxy